### ISSUE TITLE
Connection factory profiler name can fail

### ISSUE THREAD

**Issue by carl-berg** [mattiasnordqvist/Galactic-Waste-Management#39]
When running GWM, targeting `netstandard2.0` and running in `net472` in integration tests i ran into a problem in connection factory; `Assembly.GetEntryAssembly()` returned null. I hacked my way around this by replacing that with
```csharp
var assemblyName = Assembly.GetEntryAssembly()?.GetName().ToString() ?? "Unknown Assembly"
MiniProfiler mp = new MiniProfiler($"{assemblyName} - DatabaseUpdater", MiniProfiler.DefaultOptions);;
```
... but you may want to do a fancier fix of that issue.

**Comment by carl-berg** [comment]
Is there a way to use this profiler somehow? Could maybe be nice to output something if a test fails... i dunno

**Comment by mattiasnordqvist** [comment]
What kind of tests are you talking about?

**Comment by carl-berg** [comment]
An xUnit test targeting net472

**Comment by carl-berg** [comment]
Not sure really what I mean regarding profiler, I'll have to think it over. I think I've seen since cases where migrations fail but hard to see what fails. Can't remember at the top of my head though.

**Comment by carl-berg** [comment]
Should i add the fix as described above @mattiasnordqvist, or do you want to solve it another way?

**Comment by mattiasnordqvist** [comment]
We can go with this to begin with, it's not that important! :)

**Comment by carl-berg** [comment]
Yeah, i didn't think so either. I've solved it in #44 the easiest way possible. Thinking code readability in this case outweighs profiler naming.
